packages/core/src/modules/credentials/protocol/v1/V1CredentialService.ts

```diff
@@ -140,8 +140,19 @@ export class V1CredentialService extends CredentialService {
 
     const options = { ...config }
 
+    // Create record
+    const credentialRecord = new CredentialExchangeRecord({
```

Comment:
Is this really the best way to create a `CredentialExchangeRecord`? I am not the biggest fan of passing empty strings, such as with `threadId`, and passing empty lists, like with `credentials` and credentialAttributes`. These should default to these values and do not have to be passed in.